### PR TITLE
Feature/lightmaps

### PR DESCRIPTION
**Pull request by yalania**
- Added new sampler for lightmaps to materials editor
- Added new light calculation for lightmaps and new shader for completely static objects
- Support for multiple texture UV channels when importing meshes

### CODE REVIEW

File: Game/Resources/Shaders/lightmap.fs [hunk 1]
```diff
@@ -0,0 +1,218 @@
+#version 430 core
```
Comment (raulgonzalezupc) [inline Game/Resources/Shaders/lightmap.fs:1]: I don't find the difference between this shader and blinn phong

Comment (Misarte) [reply Game/Resources/Shaders/lightmap.fs:1]: Lightmap.fs calculates with fragment_normal or just a normal if there's no material, when this is erased from blinn_phong.fs, however i'm not sure what's the difference, regarding this.

Comment (yalania) [reply Game/Resources/Shaders/lightmap.fs:1]: Yeah, Lightmap.fs just ignore all the lights sources but the one coming from the lightmap

**Review by Misarte — APPROVED** [review]

File: Game/Resources/Shaders/blinn_phong.vs [hunk 2]
```diff
@@ -67,17 +68,18 @@ void main()
 
 //Skinning
 	mat4 skinning_matrix = mat4(0);
-    for(uint i=0; i<4; i++)
+    for(uint i=0; i<num_joints; i++)
```
Comment (raulgonzalezupc) [inline Game/Resources/Shaders/blinn_phong.vs:71]: better ^^

**Review by raulgonzalezupc — APPROVED** [review]

**Pull request merged**